### ISSUE TITLE
Edge Case BUG: Paragraph Emitted Twice

### ISSUE THREAD

**Issue by tajmone** [pml-lang/converter#44]
I've come across a very odd bug with PML 1.5.0...

The following source:

```pml
[doc title = PML BUGS Test \
    authors = Tristano Ajmone \
    date = 2021-06-26

[ch title = "Miscellanea" \
    id = chapter_id
    Hello



]

]
```

will produce an HTML doc where the `Hello` paragraph is shown twice in a row — but only if:

- There's no blank line between the `id =` line and "Hello" line.
- There are at least 3 blank lines between the "Hello" line and the closing `]`.

**Comment by pml-lang** [comment]
I was able to reproduce the bug. Very strange, indeed.
I tried the same example in the new version 2.0 I'm working on (using the new pXML parser), and it is rendered correctly, with just one "Hello" line.
After publishing PML 2.0 I will try again and close the bug.

**Comment by tajmone** [comment]
> Very strange, indeed.

Some bugs are just mysterious, which also makes them fascinating and challenging at the same time.

> I tried the same example in the new version 2.0 I'm working on (using the new pXML parser), and it is rendered correctly

Well, then it's not worth investigating it any further, since it will self resolve.

**Comment by pml-lang** [comment]
This bug has been fixed in version 2.0.0
